themes/ohcoe/layouts/partials/video.html

```diff
@@ -0,0 +1,6 @@
+{{ $videoId := .  }}
+<div class="video">
```

Comment:
Consider using Bootstrap's `embed-responsive` here to ensure proper viewing on all devices.

Reply:
@sdreher I did not know about that, thanks!